manifests/i/IrfanSkiljan/IrfanView/PlugIns/4.66/IrfanSkiljan.IrfanView.PlugIns.installer.yaml

```diff
@@ -0,0 +1,22 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: IrfanSkiljan.IrfanView.PlugIns
+PackageVersion: '4.66'
+InstallerType: exe
+InstallerSwitches:
+  Silent: /silent
+  SilentWithProgress: /silent
+UpgradeBehavior: install
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: IrfanSkiljan.IrfanView
+Installers:
+- Architecture: x86
+  InstallerUrl: https://files03.tchspt.com/down/iview466_plugins_x64_setup.exe
+  InstallerSha256: 080FD4E30F5CDE44B52E85425E3E406F1BBBD9FF0226DCEC7D97BE813AAC84AB
+- Architecture: x64
+  InstallerUrl: https://files03.tchspt.com/down/iview466_plugins_setup.exe
+  InstallerSha256: C779C0C8307F90CDD798377C516C286ACDE6B9843AD84ED07B19D615248B605A

```

Comment:
The installer URLs look mixed up?

Reply:
You're right, I've now corrected the URLs.